tracer/src/Datadog.Trace/AppSec/Waf/ReturnTypes.Managed/DiagnosticResultUtils.cs

```diff
@@ -143,6 +143,8 @@ public WafStats()
     }
 
     public int Total => (Failed + Loaded + Skipped);
+
+    public bool HasErrors => Errors is { Count: > 0 } || Warnings is { Count: > 0 };
```

Comment:
So this is a behaviour change from the above, because it now includes warnings, is that correct?